src/index.js

```diff
@@ -88,6 +90,8 @@ export class RootComponent extends React.Component {
 				initialHtmlModeEnabled={ initialHtmlModeEnabled }
 				initialTitle={ initialTitle }
 				postType={ postType }
+				colors={ colors }
```

Comment:
Hmmm, it's not important but I'm wondering if we need to create a consts for `colors` and `gradients`. We could use directly `this.props.colors/gradients`. Or, maybe we could do something like:

```js
const {
  initialHtmlModeEnabled,
  initialTitle = 'Welcome to Gutenberg!',
  postType = 'post',
  colors,
  gradients,
} = this.props;

let initialData = this.props.initialData;
```

Reply:
I really only went with the const originally because I wasn't sure if I'd need to default them to something. You're right though it would be cleaner if I just directly used `this.props.colors/gradients`

Reply:
Make sense :) Thanks!